05_testing_and_ci/automation_exercise.md

```diff
@@ -0,0 +1,71 @@
+
+- Add a [GitHub workflow status badge](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge) to the README that is based on your workflow and the main branch.
+    - Label this badge with "SSE CI/CD".
+
+- Submit your solution via an issue. You cannot submit a pull request since you cannot use a fork in the exercise. You can also add the issue before you are done in order to ask questions. If you are opening an issue before you are finished, please add comment in the issue mentioning that it is "work in progress". When you are finished, please add another comment that the issue (or rather your repository) is "ready for review".
+    - Create an issue in the [automation exercise repository (TODO: Update link)](https://github.com/Simulation-Software-Engineering/automation-exercise-preparation).
+    - The issue should be named `[USERNAME] GitHub Actions exercise`, e.g., `[jaustar] GitHub Actions exercise`. Please use the GitLab username here.
+    - Please add a link to your GitHub repository.
+
+## GitLab CI/CD
+
+In this subtask you should implement the workflow using GitLab and add a GitLab Runner to your repository.
+
+- Create a **fork** of the GitLab repository [automation exercise repository (TODO: Update link)](https://gitlab-sim.informatik.uni-stuttgart.de/sse-test-group/automation-exercise).
+- Install GitLab Runner on your machine and register a runner for your repository/fork. Ideally, it should be a runner that can work with the `docker` executor and is able to run Linux images. You can find the [detailed installation instructions](https://docs.gitlab.com/runner/install/). It is usually easiest to run install GitLab Runner as [Docker container](https://docs.gitlab.com/runner/install/docker.html).
+
+  If you cannot add such a runner on your machine, please sent an email with the registration token and your GitLab username to [alexander.jaust@ipvs.uni-stuttgart.de](mailto:alexander.jaust@ipvs.uni-stuttgart.de). We will add a runner to your repository/fork.
+
+- Set up the GitLab pipeline to run the checks, tests, and to create the coverage report.
+    - The pipeline should have three jobs (`style_check`, `test`, `coverage_report`). Each of these jobs should carry out the corresponding step described in [Task descriptions](## Task descriptions).
+    - The intermediate coverage information (`.coverage`) should be kept for [three hours](https://docs.gitlab.com/ee/ci/yaml/index.html#artifactsexpire_in).
+    - Make sure that your runner is online while you make changes to the repository. Otherwise the CI/CD pipeline cannot run.
+    - You may use the Docker image `ajaust/automation-exercise` (based on Ubuntu). You can find its Dockerfile in the GitLab repository (TODO: Add Dockerfile.).
```

Comment:
Better say that they should use the image from Dockerhub and just for reference, they also find the corresponding Dockerfile in the repo.